src/services/HoroscopeService.ts

```diff
@@ -1,20 +1,33 @@
 import { constants } from '@klasa/timestamp';
 import { load as cheerio } from 'cheerio';
-import fetch from 'node-fetch';
+import fetch, { RequestInit } from 'node-fetch';
 import HoroscopeEntry from '../structures/HoroscopeEntry';
 import type days from '../utils/days';
 import type sunsigns from '../utils/sunsigns';
 import { toTitleCase } from '../utils/utils';
 
 export default class HoroscopeService {
 	#baseSeletor = '.sign-hero__horoscope-wrapper > .day-tabs-content--sign-page';
+	#fetchOptions: RequestInit = {
+		headers: {
+			accept: '*/*',
+			'accept-encoding': 'gzip, deflate, br',
+			'accept-language': 'en-US,en;q=0.9,nl;q=0.8',
```

Comment:
Based on https://www.w3.org/International/O-charset-lang.html, it'd make sense if the secondary preferred language were German, since the VPS is hosted there.
```suggestion
			'accept-language': 'en-US,en;q=0.9,de;q=0.8',
```